### DISCUSSION TITLE
Could I pull messages from another email service from my side?

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [docker-mailserver/docker-mailserver#2631] [Q&A] [answered]
I have another email address $\beta$ actively used for academic purposes. 
But sadly, the email service of $\beta$ does not have an automatic forwarding feature.
Is there any way to pull emails from $\beta$ to my "docker-mailserver"?

**Comment by casperklein** [comment, marked as answer]
You are probably looking for something like [fetchmail](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/mail-fetchmail/).

**Reply by WantToLearnJapanese** [reply]
Thanks, this is precisely the solution I was looking for!
